<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AddRoadDialog</class>
    <widget class="QDialog" name="AddRoadDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>140</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Добавление дороги</string>
        </property>
        <widget class="QDialogButtonBox" name="buttonBox">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>100</y>
                    <width>360</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="orientation">
                <enum>Qt::Horizontal</enum>
            </property>
            <property name="standardButtons">
                <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="geometry">
                <rect>
                    <x>200</x>
                    <y>10</y>
                    <width>190</width>
                    <height>40</height>
                </rect>
            </property>
            <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <property name="maximum">
                <double>100.000000000000000</double>
            </property>
            <property name="value">
                <double>0.000000000000000</double>
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>190</width>
                    <height>40</height>
                </rect>
            </property>
            <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Введите значение:</string>
            </property>
            <property name="margin">
                <number>10</number>
            </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>60</y>
                    <width>360</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Двунаправленная дорога</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>AddRoadDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>AddRoadDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
